packages/superset-ui-number-format/src/index.js

```diff
@@ -0,0 +1,10 @@
+import * as NumberFormats from './NumberFormats';
+
+export {
+  default as getNumberFormatterRegistry,
+  formatNumber,
+  getNumberFormatter,
+} from './NumberFormatterRegistrySingleton';
+
+export { default as NumberFormatter, PREVIEW_VALUE } from './NumberFormatter';
+export { NumberFormats };
```

Comment:
you might be able to `export { * as NumberFormats } from './NumberFormats'`

Reply:
Hmm.. Tried and didn't work.